RadixWallet/Features/ProfileBackupsFeature/RestoreProfileFromBackup/Children/ImportSeedPhrasesFlow/ImportMnemonicsFlowCoordinator.swift

```diff
@@ -255,17 +255,11 @@ public struct ImportMnemonicsFlowCoordinator: Sendable, FeatureReducer {
 			return .none
 		} else {
 			state.destination = nil
-			return .run { [skipped = state.skipped, imported = state.imported, newMainBDFS = state.newMainBDFS, context = state.context] send in
-				if let newMainBDFS, !context.isFromOnboarding {
-					try await factorSourcesClient.saveNewMainBDFS(newMainBDFS)
```

Comment:
new BDFS will be [saved by Sargon](https://github.com/radixdlt/sargon/blob/6512769d46a954b1c16cf6d06bf6e921e567098b/crates/sargon/src/system/sargon_os/sargon_os.rs#L142)